www/include/reporting/dashboard/template/viewServicesLog.ihtml

```diff
@@ -95,7 +93,7 @@
 							<td class='ListColCenter'>{$summary.UNDETERMINED_TF}</td>
 							<td class='ListColRight'>{$summary.UNDETERMINED_TP}%</td>
 							<td class='ListColRight'></td>
-							<td class='ListColRight'>{$summary.UNDETERMINED_A}</td>
```

Comment:
`UNDETERMINED_A` is never set ?

Reply:
Only in viewServiceLog.php line 155 but it is set to null (I had a warning concerning this parameter so I deleted)

Reply:
I think it should be better to get this value here in getServicesLogs https://github.com/centreon/centreon/blob/develop/www/include/reporting/dashboard/DB-Func.php#L461

In this request, with a sum as it is done for other status.